vizro-core/src/vizro/figures/_kpi_cards.py

```diff
@@ -129,6 +135,11 @@ def kpi_card_reference(  # noqa: PLR0913
         >>> vm.Page(title="Page", components=[vm.Figure(figure=kpi_card_reference(...))])
 
     """
+
+    def _get_footer_class(delta: float, reverse_color: bool) -> str:
```

Comment:
```suggestion
    def _get_footer_class(delta: float) -> str:
```

`reverse_color` will still be defined inside the closure without it being an explicit function argument.

Reply:
Given this is meant to be a relatively simple copy and pastable function I would even go one step further and not use a closure here at all and instead just do:
```
pos_color, neg_color = ("color-neg", "color-pos") if reverse_color else ("color-pos", "color-neg")
footer_class = pos_color if delta > 0 else neg_color if delta < 0 else ""
```